### PR TITLE
website: Add security page

### PR DESCRIPTION
**Pull request by sethvargo**

### CODE REVIEW

File: website/source/security.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+description: |-
+  Terraform takes security very seriously. Please responsibly disclose any security vulnerabilities found and we'll handle it quickly.
+---
+
+<h1>Terraform Security</h1>
+
+<p>
+  We understand that many users place a high level of trust in HashiCorp
+  and the tools we build. We apply best practices and focus on security to
+  make sure we can maintain the trust of the community.
```
Comment (jamtur01) [inline website/source/security.html.erb]: This statement is kinda empty? Perhaps list some practices? We:

* We code review all changes for security concerns.
* We use and require encryption when possible.
* We publish secure packages and checksums for binaries.
* We run a security bounty program...

And other stuff that is relevant.

Or link to a central page that lists central Hashicorp security practice...

File: website/source/security.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+</p>
+
+<p>
+  We deeply appreciate any effort to disclose vulnerabilities responsibly
```
Comment (jamtur01) [inline website/source/security.html.erb]: Period is after the closing p.

File: website/source/security.html.erb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+<p>
+  If you would like to report a vulnerability, please see the <a
+  href="https://www.hashicorp.com/security.html">HashiCorp security
+  page</a>, which has the proper email to communicate with as well as our
+  PGP key. Please <strong>do not create an GitHub issue for security
+  concerns</strong>.
+</p>
+
+<p>
+  If you are not reporting a security sensitive vulnerability, please open
```
Comment (jamtur01) [inline website/source/security.html.erb]: I wouldn't use a negative here - this statement is confusing - perhaps:

`If you need to report a non-security related bug, please open...`

File: website/source/security.html.erb (hunk 1 above)
Comment (sethvargo) [inline website/source/security.html.erb]: Hey James - that's good feedback. This is a general page we are putting across all our projects, but I'll take these bullet points and send them to our team to add to https://www.hashicorp.com/security.html, which is linked below.

**Pull request merged**